<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control de Asistencias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#eef7f7;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 7px;
border: 2px solid #58e4f2;
background-color: rgb(198, 206, 206);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeNowLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Archivo SemiBold</family>
      <pointsize>1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
    </property>
    <property name="text">
     <string>DD/MM//YYYY HH:MM:SS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>81</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>511</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>35</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string>Nombre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="codeLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string>Código Matrícula:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lastAttendanceLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string>Ult. Asistencia:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="codeLabel_2">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string>N° Asistencias:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="codeLabel_3">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color: #31c8d8;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string>Nro Faltas:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>35</number>
       </property>
       <item>
        <widget class="QLabel" name="actualNameLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 15px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color:#83e5ef;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="actualCodeLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 15px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color:#83e5ef;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="actualLastAttendanceLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 15px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color:#83e5ef;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="actualNumberOfAttendanceLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 15px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color:#83e5ef;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="actualNumberOfNoAttendanceLabel">
         <property name="font">
          <font>
           <family>Archivo SemiBold</family>
           <pointsize>1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 15px;
font-family: Archivo SemiBold;
font-weight: bold;
color:#272d2d;
background-color:#83e5ef;
padding: 3px;
border-radius: 7px;
border: 2px solid #58e4f2;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>-10</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
    margin: 0;
    padding: 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img.qrc">:/logomecat/logomecat_resized.png</pixmap>
    </property>
   </widget>
   <zorder>line</zorder>
   <zorder>imageLabel</zorder>
   <zorder>timeNowLabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>logoLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionGenerarRegistroDiario"/>
    <addaction name="actionRegistrarAlumno"/>
    <addaction name="actionExportar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExportar">
   <property name="text">
    <string>Exportar</string>
   </property>
   <property name="iconText">
    <string>Exportar</string>
   </property>
   <property name="toolTip">
    <string>Exportar registro de asistencias</string>
   </property>
   <property name="statusTip">
    <string>Exportar registro de asistencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir de Aplicación</string>
   </property>
   <property name="statusTip">
    <string>Salir de Aplicación</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRegistrarAlumno">
   <property name="text">
    <string>Registrar Alumno</string>
   </property>
   <property name="toolTip">
    <string>Registrar Alumno Nuevo</string>
   </property>
   <property name="statusTip">
    <string>Registrar Alumno Nuevo</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionGenerarRegistroDiario">
   <property name="text">
    <string>Generar Registro Diario</string>
   </property>
   <property name="statusTip">
    <string>Generar Registro Diario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
